src/PageComponents/ContentContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
```

Comment:
Also if you guys are gonna componentize your form even further, feel free to use TypeScript to type your props for each one - that is if you need props in the first place.